src/reconcile/rec_child.c

```diff
@@ -33,9 +33,12 @@ __rec_child_deleted(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_REF *ref, WT_C
      * A visible update to be in READY state (i.e. not in LOCKED or PREPARED state), for truly
      * visible to others.
      */
-    if (F_ISSET(r, WT_REC_VISIBILITY_ERR) && page_del != NULL &&
-      __wt_page_del_active(session, ref, false))
-        WT_PANIC_RET(session, EINVAL, "reconciliation illegally skipped an update");
+    if (F_ISSET(r, WT_REC_EXPECT_CLEAN | WT_REC_VISIBILITY_ERR) && page_del != NULL &&
```

Comment:
I generally understand what this part of the code is doing but not very sure. Please check that I'm handling the new flag correctly.

Reply:
Who would you like to check this?

Reply:
I'm not confident about this. Anyone knows about this part of the code.

Reply:
I think this is okay.  Generally, if you make the new flag do exactly the same checks as `WT_REC_VISIBILITY_ERR` but the new flag returns `EBUSY` rather than causing a panic, I think it will be correct.